src/platform/sdl/sdl2_audio.cpp

```diff
@@ -22,9 +22,15 @@
 #include <cassert>
 #include <cstdint>
 #include <chrono>
+#ifndef __PS4__
 #include <SDL.h>
 #include <SDL_audio.h>
 #include <SDL_version.h>
+#else
```

Comment:
How old is the SDL2 configuration for PS4? Likely this is not needed, since the paths are provided.